backend/telemetry_core/src/state/chain.rs

```diff
@@ -53,6 +54,8 @@ pub struct Chain {
     timestamp: Option<Timestamp>,
     /// Genesis hash of this chain
     genesis_hash: BlockHash,
+    /// Maximum number of nodes allowed in this chain
```

Comment:
```suggestion
    /// Maximum number of nodes allowed to connect from this chain
```

"in" sounds a bit like it's a cap on the number of allowed nodes participating in a network?

Reply:
That's fair :)